homeassistant/brands/yale_us_ca.json

```
@@ -0,0 +1,5 @@
+{
+  "domain": "yale_us_ca",
+  "name": "Yale (US/Canada)",
+  "integrations": ["august", "yalexs_ble"]
```

```suggestion
  "integrations": ["august", "august_ble"]
```

We actually have an `august_ble` virtual so I think we can use that